scripts/collins.sh

```diff
@@ -32,7 +32,18 @@ JMX_OPTS="-Dcom.sun.management.jmxremote -Dcom.sun.management.jmxremote.port=333
 DEBUG_OPTS="-XX:ErrorFile=${LOG_HOME}/$APP_NAME/java_error%p.log -XX:+HeapDumpOnOutOfMemoryError -XX:HeapDumpPath=/tmp/collinsDump.hprof"
 
 # Check for config overrides
-[ -f /etc/sysconfig/collins ] && . /etc/sysconfig/collins
+DISTRO=$(lsb_release -d | awk '{print $2}')
+
+case $DISTRO in
+    Ubuntu|Debian)
```

Comment:
this should probably be default cause honestly, cause other OSes (arch, etc) use this pattern as well

Reply:
I realized: Why do I need to check OS version at all?  Just source those files if they exist, right?

Reply:
👍